.github/issue_template/question.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Asking a question / requesting help
+about: Ask the community for help or advise
+title: '[Question] '
+labels: 'Question'
```

Comment:
Questionlabel should be lower case for consistency and to match the labels in the repo

Reply:
All seems fie to me, although I am not clear how these templates work. Do I need to do anything now?

Reply:
changed to lower case in https://github.com/fNIRS/snirf/pull/138/commits/6d16f1caa2ed5186c5f5597266c1f08fa3e9997c

Reply:
@dboas you (and Sam) just need to approve the PR to allow me to merge it